packages/components/src/components/post-user-menu/post-user-menu.tsx

```diff
@@ -0,0 +1,88 @@
+import { Component, Host, h, Prop, Watch, Element } from '@stencil/core';
+import { nanoid } from 'nanoid';
+import { version } from '@root/package.json';
+import { checkRequiredAndType } from '@/utils';
+
+export interface User {
+  name: string;
+  surname?: string;
+  email?: string;
+}
+
+@Component({
+  tag: 'post-user-menu',
+  styleUrl: 'post-user-menu.scss',
+  shadow: true,
+})
+export class PostUserMenu {
+  private readonly menuId = `p${nanoid(11)}`;
```

Comment:
I'm pretty sure this will cause a hydration error.
It would be great if you could check that. If it's the case, we need to find a general solution for this problem, because I totally see the need for that sometimes.

Reply:
Do you have any insight for that?